### ISSUE TITLE
Rows are failing to filter

### ISSUE THREAD

**Issue by bennypowers** [vaadin/vaadin-grid#594]
```html
<!DOCTYPE html>

<link rel="import" href="../bower_components/polymer/polymer.html">
<link rel="import" href="../bower_components/app-layout/app-drawer/app-drawer.html">
<link rel="import" href="../bower_components/app-layout/app-drawer-layout/app-drawer-layout.html">
<link rel="import" href="../bower_components/app-layout/app-header/app-header.html">
<link rel="import" href="../bower_components/app-layout/app-header-layout/app-header-layout.html">
<link rel="import" href="../bower_components/app-layout/app-scroll-effects/app-scroll-effects.html">
<link rel="import" href="../bower_components/app-layout/app-toolbar/app-toolbar.html">
<link rel="import" href="../bower_components/marked-element/marked-element.html">
<link rel="import" href="../bower_components/polymerfire/firebase-query.html">
<link rel="import" href="../bower_components/paper-dialog/paper-dialog.html">
<link rel="import" href="../bower_components/iron-icon/iron-icon.html">
<link rel="import" href="../bower_components/paper-dialog-scrollable/paper-dialog-scrollable.html">
<link rel="import" href="../bower_components/paper-icon-button/paper-icon-button.html">
<link rel="import" href="../bower_components/paper-fab/paper-fab.html">
<link rel="import" href="../bower_components/vaadin-grid/vaadin-grid.html">
<link rel="import" href="../bower_components/vaadin-grid/vaadin-grid-filter-behavior.html">
<link rel="import" href="shared-styles.html">
<link rel="import" href="ksybf-icons.html">
<link rel="import" href="ksybf-edit.html">

<dom-module id="ksybf-staff">
  <template>


    <firebase-auth id="auth" user="{{user}}" provider="google" on-error="handleError"></firebase-auth>

    <firebase-query id="membersQuery"
        path="/members"
        data="{{members}}"
    ></firebase-query>

        <main id="staff">
          <template is="dom-if" if="[[user]]" restamp="true">
            <vaadin-grid items="[[members]]">
              <vaadin-grid-column frozen width="10em">
                <template class="header">Member ID</template>
                <template>[[item.id]]</template>
              </vaadin-grid-column>
              <vaadin-grid-column width="48px">
                <template class="header">
                  <iron-icon icon="photo"></iron-icon>
                </template>
                <template>
                  <iron-image src="[[item.photo]]" preload width="48" sizing="cover"></iron-image>
                </template>
              </vaadin-grid-column>
              <vaadin-grid-column>
                <template class="header">
                  <vaadin-grid-filter path="lastName">
                    <paper-input no-label-float label="Last Name"></paper-input>
                  </vaadin-grid-filter>
                </template>
                <template>[[item.lastName]]</template>
              </vaadin-grid-column>
              <vaadin-grid-column>
                <template class="header">
                  <vaadin-grid-filter path="firstName">
                    <paper-input no-label-float label="First Name"></paper-input>
                  </vaadin-grid-filter>
                </template>
                <template>[[item.firstName]]</template>
              </vaadin-grid-column>
              <vaadin-grid-column>
                <template class="header">
                  <vaadin-grid-filter path="email">
                    <paper-input no-label-float label="Email Address"></paper-input>
                  </vaadin-grid-filter>
                </template>
                <template>[[item.email]]</template>
              </vaadin-grid-column>
              <vaadin-grid-column>
                <template class="header">
                  <vaadin-grid-filter path="custom">
                    <paper-input no-label-float label="Custom Paragraph"></paper-input>
                  </vaadin-grid-filter>
                </template>
                <template>
                  <marked-element markdown="[[item.custom]]"></marked-element>
                </template>
              </vaadin-grid-column>
            </vaadin-grid>
          </template>

  </template>

  <script>
    Polymer({
      is: 'ksybf-staff'
    });
  </script>
</dom-module>

```
When using this code, rows do not filter. The same is true when paper-input is replaced with `input`

**Comment by Saulis** [comment]
Hi!

Are you on `2.0.0-pre.2` on the very bleeding edge (`2.0-dev` branch) ?

We've just yesterday merged #593 which will change the behaviour so that you need to manually bind the filtering value from `vaadin-grid-filter` to your custom input.

**Comment by bennypowers** [comment]
Confirm that binding values in that way works on 2.0-dev. Thanks!

```html
<vaadin-grid-column>
  <template class="header">
    <vaadin-grid-filter path="firstName" value="{{_firstNameFilter}}">
      <paper-input no-label-float label="First Name" value="{{_firstNameFilter}}"></paper-input>
    </vaadin-grid-filter>
  </template>
  <template>[[item.firstName]]</template>
</vaadin-grid-column>
```
